app/controllers/users.js

```diff
@@ -338,6 +339,46 @@ exports.addDonation = (req, res) => {
   res.send();
 };
 
+exports.addDonation = (req, res) => {
+  if (req.body && req.user && req.user._id) {
+    // Verify that the object contains crowdrise data
+    if (req.body.amount && req.body.crowdrise_donation_id && req.body.donor_name) {
+      User.findOne({
+        _id: req.user._id
+      })
+        .exec((err, user) => {
+        // Confirm that this object hasn't already been entered
+          let duplicate = false;
+          for (let i = 0; i < user.donations.length; i++) {
+            if (user.donations[i].crowdrise_donation_id === req.body.crowdrise_donation_id) {
+              duplicate = true;
+            }
+          }
+          // if (!duplicate) {
```

Comment:
Remove commented out code here and every other place
@Daramola98

Reply:
Ok